File: FirstUniqueCharacter.py
class Solution:
    def firstUniqChar(self, s: str) -> int:
        frequency = [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
        for j in range (len(s)):
            frequency[ord(s[j])-ord('a')] += 1

        for j in range (len(s)):
            if (frequency[ord(s[j])-ord('a')] == 1):
                return j;
        
        return -1;

File: FirstUniqueCharacter.cpp
class Solution {
public:
    int firstUniqChar(string s) {
        vector <int> frequency(26,0);
        for(int i=0; i< s.length(); ++i){
            frequency[s[i]-'a'] += 1;
        }
        for(int i=0; i< s.length(); ++i){
            if (frequency[s[i]-'a'] == 1)
                return i;
        }
        return -1;
    }
};
